configure.ac

```diff
@@ -718,8 +720,27 @@ AC_MSG_NOTICE([checking required package dependencies])
 # NL_WITH_PACKAGE(...)
 
 #
+# Checks for missing functions
+#
+
+AC_DEFUN([CHECK_MISSING_FUNC], [
+    AC_CHECK_FUNC($1, [], [
+        nl_cv_missing_$1=yes
+        MISSING_CPPFLAGS="${MISSING_CPPFLAGS} "'-include ${top_srcdir}/src/missing/$1/$1.h'
+        MISSING_LIBADD="${MISSING_LIBADD} "'${top_builddir}/src/missing/$1/lib$1.la'
+    ])
+    AM_CONDITIONAL(m4_toupper(MISSING_$1), [test "${nl_cv_missing_$1}" = "yes"])
+])
+AC_SUBST(MISSING_CPPFLAGS)
+AC_SUBST(MISSING_LIBADD)
+
+CHECK_MISSING_FUNC([strlcpy])
+CHECK_MISSING_FUNC([strlcat])
```

Comment:
Hmm, I was just looking at [strlcpy](https://linux.die.net/man/3/strlcpy) and I see that it's return is different. It returns a count, while [strcpy_s](https://msdn.microsoft.com/en-us/library/td1esda9.aspx?f=255&MSPPError=-2147217396) return success or failure.